### ISSUE TITLE
Dual screen, its not on the other display

### ISSUE THREAD

**Issue by PGTBoos** [White-Tiger/T-Clock#236]
Hi, i really like T-Clock 
But i have a dual screen setup on my laptop, is it somehow possible to have it on both screens ?.
Currently its only on screen 1 (my laptop) the external display/taskbar doesnt show it.
Running it on windows 10

**Comment by OliverGraetsch** [comment]
Did you check the option "Duplicate clock on each additional taskbar" in "Miscellaneous"-tab in T-Clock Options?

**Comment by Cleridwen** [comment]
I confirm it does go away on my secondary monitors. It is still present on my main screen, but on the other screens, there's only a blank space where the clock should be.
Sometimes, it is still there but kind of invisible: it opens the calendar when I click the empty space. Sometimes it's completely missing. Sometimes it is still there on only one of the secondary monitors.

I believe this happens whenever the resolution of a monitor changes (could be due to a full screen game), a monitor is plugged or unplugged, the displays are rearranged in Windows's settings, and potentially also when putting the PC to sleep (STR) mode or hibernating.

This needs more debugging, because I'm not sure what exactly is the cause and the fix for that. Restarting T-Clock doesn't make the clock go back on the secondary monitors, restarting my PC does put the clocks back on every taskbar. 

Monitors: main 3440x1440 100% scaling, 1620x1080 100%, 1920x1080 125%, 800x600 100%
_My specs if needed: https://pastebin.com/XDvYsDEy_
_Additional info: Windows 10 20H2, "small taskbar buttons" turned on_

**Comment by ghost** [comment]
Yes I con confirm I have the same problem as @hydargos123
Multiple screens with different resolutions

**Comment by thatrossachap** [comment]
I have a similar issue, but that, between the two screens (a 4K laptop display and a 1080p monitor), the scaling changes, with my monitor clock far oversized, as it seems to scale according to the primary display.

**Comment by Cleridwen** [comment]
@thatrossachap yep, I have the scaling issue too. One of my secondary monitor is at 125% on Windows, the others are at 100%, and the clock on that monitor appears smaller.
Too bad that this is not getting bugfixes or updates anymore. There is no other Windows taskbar clock replacement as complete as that one.
